core/layouts/geary/layout--geary.tpl.php

```diff
@@ -31,82 +31,93 @@
 
   <?php if ($content['header']): ?>
     <header class="l-header" role="banner" aria-label="<?php print t('Site header'); ?>">
-      <div class="container container-fluid">
-        <div class="row">
-          <?php print $content['header']; ?>
-        </div><!-- /.row -->
+      <div class="l-header-inner container container-fluid">
+        <?php print $content['header']; ?>
       </div><!-- /.container -->
     </header>
   <?php endif; ?>
 
-  <div class="l-wrapper">
-    <div class="l-wrapper-inner container container-fluid">
+  <main class="l-wrapper">
+    <section class="l-wrapper-inner">
```

Comment:
Recommend wrapping every page in a `section` element so we can use the `header` tag for the `h1`.